src/list-names.py

```diff
@@ -1,17 +1,14 @@
-import argparse
-
-
-def main():
-    argparser = argparse.ArgumentParser(
-        description="Extract the names from a simple-fastq file")
-    argparser.add_argument(
-        "fastq",
-        type=argparse.FileType('r')
-    )
-    args = argparser.parse_args()
-
-    print(f"Now I need to process the records in {args.fastq}")
-
-
-if __name__ == '__main__':
-    main()
+import sys
+
+def list_names():
+    # load input:
+    inFile = sys.argv[1]
+    with open(inFile,'r') as j:
```

Comment:
Most programmers on this planet would expect a variable named `j` to hold an integer (unless in a specific domain where `j` has some other interpretation). You are making it hard for others (and your future self) by naming an open file `j`.